TimberWinR/Inputs/IISW3CInputListener.cs

```diff
@@ -145,6 +145,12 @@ private void IISW3CWatcher(string location)
                         catch (Exception ex)
                         {
                             LogManager.GetCurrentClassLogger().Error(ex);
+                            try
```

Comment:
Why add the sleep in the catch here but in the finally in WindowsEvtInputListener? Having the wait in the finally seems safer...